docs/analytics.eventpayload.entity.md

```diff
@@ -0,0 +1,17 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [@yext/analytics](./analytics.md) &gt; [EventPayload](./analytics.eventpayload.md) &gt; [entity](./analytics.eventpayload.entity.md)
+
+## EventPayload.entity property
+
+The Yext entity to which the event corresponds.

```

Comment:
Can we describe what the entityId vs entityUID is?

Reply:
I'll just rebase off your change once it's up

Reply:
Use this when rebasing to avoid having others changes show up in your diff. 

https://stackoverflow.com/questions/18323665/pull-request-on-github-showing-commits-rebased-from-master